CMakeLists.txt

```diff
@@ -148,6 +148,47 @@ target_link_libraries(d8
     v8_libplatform
 )
 
+# add shell and hello-world
+add_executable(
+        shell
+        v8/samples/shell.cc
+)
+target_include_directories(shell
```

Comment:
Is there a trick to resue `target_include_directories` and `target_link_libraries` ? 

cc @bnoordhuis

Reply:
I will try this in a followup PR.